VideoLocker/src/main/java/org/edx/mobile/util/images/ErrorUtils.java

```diff
@@ -35,4 +40,34 @@ public static String getErrorMessage(@NonNull Throwable ex, @NonNull Context con
         }
         return errorMessage;
     }
+
+    /**
+     * Deliver the appropriate error message to their callbacks.
+     *
+     * @param ex The exception that occurred
+     * @param context A Context
```

Comment:
These are phrases, not sentences, so not having a period should be fine. I'll add them anyway as part of my review changes. Care to expand on what's weird about the formatting?